### ISSUE TITLE
`@phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint` seems not working

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1346]
It is mentioned in docs, but

```php
<?php declare(strict_types = 1);

namespace Tests\phpcs;

interface A {
    /**
     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint
     *
     * @return void
     */
    public function a();
}
```

gives

```
$ ./vendor/bin/phpcs ./tests/phpcs/A.php

FILE: /project/backend/tests/phpcs/A.php
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 11 | ERROR | [x] Method \Tests\phpcs\A::a() does not have native return type hint for its return value but it should be possible to add it based on @return annotation "void".
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 142ms; Memory: 10MB
```

```xml
<?xml version="1.0"?>
<ruleset 
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint"/>
</ruleset>
```

_PS: v7.0.19_

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/b480060d43a135de4fc51bf2fa3b0148c3f74462
